src/templates/profile.ts

```diff
@@ -15,3 +16,53 @@ ${name} usecase
 usecase ${name} {}
 `;
 }
+
+export function complete(name: string): string {
+  return `
+"""
+${name} usecase
```

Comment:
```suggestion
${name} use case
```
to match docs & writing standards?


Also I'd add a second line with "usecase description blah blah", so that it's apparent it doesn't need to (and shouldn't!) be a single line description